code/game/objects/items/nemesis_items.dm

```diff
@@ -0,0 +1,458 @@
+
+/obj/item/nemesis_mine
+	name = "energy mine"
+	desc = "A high-tech trap, designed to electrocute any intruders that step on it."
+	icon = 'icons/obj/nemesis.dmi'
+	icon_state = "nemesis_trap_inactive"
+	w_class = WEIGHT_CLASS_SMALL
+
+	var/active = FALSE
+	var/overloaded = FALSE
+	var/obj/item/clothing/glasses/hud/security/sunglasses/nemesis/glasses
+
+/obj/item/nemesis_mine/Crossed(mob/living/AM)
```

Comment:
... ghosts and stuff can't trigger this, right?

Reply:
right